### ISSUE TITLE
V-Measure score metric

### ISSUE THREAD

**Issue by gcalcedo** [gcalcedo/clusview#80]
Add a new metric based on [V-Measure Score](https://scikit-learn.org/stable/modules/generated/sklearn.metrics.v_measure_score.html).
